source/Body.cpp

```diff
@@ -236,7 +240,7 @@ void Body::LoadSprite(const DataNode &node)
 			child.PrintTrace("Skipping unrecognized attribute:");
 	}
 
-	if(scale != 1.f)
+	if(scale != Point(1, 1))
```

Comment:
we seem to use this default Scale a lot, just wondering would it be appropriate to add it as a static member of the Point class to compare with instead of creating so many points?
prob fine if you dont change it but just pointing the possibility out